### PR TITLE
ci: add base image build

### LINKED ISSUE #4441: CI: let fatpack image depend on lean image
If possible, this should drastically reduce build time for the fatpack and with the addition of #4440, it should be possible to build the fatpack in the GitHub ci

### PR DESCRIPTION
**Pull request by openoms**
add stop file and --image_size 20G

the artifact to be used in: https://github.com/raspiblitz/raspiblitz/pull/4454

### CODE REVIEW

File: .github/workflows/arm64-rpi-base-image.yml [hunk 1]
```diff
@@ -0,0 +1,83 @@
+name: arm64-rpi-base-image-build
```
Comment (cstenglein) [inline .github/workflows/arm64-rpi-base-image.yml:1]: Looking good, not sure if we should build the lean / fatpack here as well with the `needs` syntax.

**Review by cstenglein — APPROVED** [review]

File: .github/workflows/arm64-rpi-base-image.yml (hunk 1 above)
Comment (openoms) [reply .github/workflows/arm64-rpi-base-image.yml:1]: I think the next step could be to use the base-image artifact built in this PR in #4454. Would only merge if the added action is useful.

Comment (openoms) [reply .github/workflows/arm64-rpi-base-image.yml:1]: ok, pulled the changes from #4454 and unifying in the `arm64-rpi-fatpack-image` action

File: .github/workflows/arm64-rpi-fatpack-image.yml [hunk 2]
```diff
@@ -72,13 +50,89 @@ jobs:
...
+            echo "GITHUB_USER=$(echo ${{github.repository}} | cut -d'/' -f1)" >> $GITHUB_OUTPUT
+          fi
+
+      - name: Checkout code
+        uses: actions/checkout@v4
+
+      - name: Download base image
+        uses: actions/download-artifact@v4
+        with:
+          name: raspiblitz-arm64-rpi-base
```
Comment (cstenglein) [inline .github/workflows/arm64-rpi-fatpack-image.yml]: The name here must be the same as the name you uploaded it with.
So `raspiblitz-arm64-rpi-image-${{ env.BUILD_DATE }}-${{ env.BUILD_VERSION }}`
But be careful that the `BUILD_DATE` & `BUILD_VERSION` are really the same

**Review by rootzoll — APPROVED** [review]
untACK

**Comment by openoms** [comment]
so far the LND, CLN (long complie), web UI and API installations ran through ok.

Joinmarket and BTCPay had problems, will try to test them separately.

The JM install issue is reproduced in so can work on it there: https://github.com/openoms/raspiblitz/pull/82

**Comment by openoms** [comment]
Reverted the expirementation with the fatpack in Actions.

Only generating the base image here as the original scope of the PR which works now.

Will continue to try to the fatpack build work in a separate PR building on the base image.

**Comment by openoms** [comment]
merging as only affecting the github workflows and ci directory

**Pull request merged**